app/partials/errorDetail.html

```diff
@@ -1,3 +1,7 @@
+<div class="page-size" ng-if="error">Showing {{start()}} - {{end()}} of {{total()}} &nbsp|&nbsp <label for="page-size" class="pagination_label">View</label>
```

Comment:
a. the HTML entity is missing the semi-colon: `&nbsp;` not `&nbsp`
b. there's rarely a valid reason to use `&nbsp;`. You should wrap the "Showing..." text in a span and use CSS to adjust the padding around it instead.